radio/src/myeeprom.h

```diff
@@ -2052,6 +2056,27 @@ enum DSM2Protocols {
   DSM2_PROTO_DSMX,
 };
 
+
+// rfProtcol is currently a signed int
+enum MultiModuleRFProtocols {
+    MM_RF_PROTO_FLYSKY=-8,
+    MM_RF_PROTO_FIRST=MM_RF_PROTO_FLYSKY,
+    MM_RF_PROTO_HUBSAN,
+    MM_RF_PROTO_FRSKY_D8,
+    MM_RF_PROTO_HISKY,
+    MM_RF_PROTO_V2X2,
+    MM_RF_PROTO_DSM2,
+    MM_RF_PROTO_DEVO,
+    MM_RF_PROTO_YD717,
+    MM_RF_PROTO_V977,
```

Comment:
Not really, I did not want to break the eeprom format and had only 4bit available.